glucose manylinux aarch64: error: '_FPU_EXTENDED' was not declared in this scope

```
 [glucose-4.2.1]   [spkg-install] /host/sage-manylinux_2_28_aarch64/var/tmp/sage/build/glucose-4.2.1/src/simp/Main.cc: In function 'int main(int, char**)':
  [glucose-4.2.1]   [spkg-install] /host/sage-manylinux_2_28_aarch64/var/tmp/sage/build/glucose-4.2.1/src/simp/Main.cc:127:46: error: '_FPU_EXTENDED' was not declared in this scope
  [glucose-4.2.1]   [spkg-install]   127 |         _FPU_GETCW(oldcw); newcw = (oldcw & ~_FPU_EXTENDED) | _FPU_DOUBLE; _FPU_SETCW(newcw);
  [glucose-4.2.1]   [spkg-install]       |                                              ^~~~~~~~~~~~~
  [glucose-4.2.1]   [spkg-install] /host/sage-manylinux_2_28_aarch64/var/tmp/sage/build/glucose-4.2.1/src/simp/Main.cc:127:63: error: '_FPU_DOUBLE' was not declared in this scope
  [glucose-4.2.1]   [spkg-install]   127 |         _FPU_GETCW(oldcw); newcw = (oldcw & ~_FPU_EXTENDED) | _FPU_DOUBLE; _FPU_SETCW(newcw);
  [glucose-4.2.1]   [spkg-install]       |                                                               ^~~~~~~~~~~
```
https://github.com/passagemath/passagemath/actions/runs/15220322862/job/42818891690#step:12:15235

--- a/build/pkgs/glucose/patches/2.patch
+++ b/build/pkgs/glucose/patches/2.patch
@@ -1,0 +1,36 @@
+From c9f3ec32c92971bd193147a98a956b6876caf1be Mon Sep 17 00:00:00 2001
+From: Petr Ambroz <petrambroz@icloud.com>
+Date: Tue, 5 Nov 2024 12:29:01 +0000
+Subject: [PATCH] fix FPU control preventing compilation on ARM CPUs
+
+---
+ parallel/Main.cc | 2 +-
+ simp/Main.cc     | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/parallel/Main.cc b/parallel/Main.cc
+index 7f224a8..00d044f 100644
+--- a/parallel/Main.cc
++++ b/parallel/Main.cc
+@@ -97,7 +97,7 @@ int main(int argc, char** argv)
+         setUsageHelp("c USAGE: %s [options] <input-file> <result-output-file>\n\n  where input may be either in plain or gzipped DIMACS.\n");
+         // printf("This is MiniSat 2.0 beta\n");
+         
+-#if defined(__linux__)
++#if defined(__linux__) && (defined(__i386__) || defined(__x86_64__))
+         fpu_control_t oldcw, newcw;
+         _FPU_GETCW(oldcw); newcw = (oldcw & ~_FPU_EXTENDED) | _FPU_DOUBLE; _FPU_SETCW(newcw);
+         printf("c WARNING: for repeatability, setting FPU to use double precision\n");
+diff --git a/simp/Main.cc b/simp/Main.cc
+index c2327df..8302179 100644
+--- a/simp/Main.cc
++++ b/simp/Main.cc
+@@ -122,7 +122,7 @@ int main(int argc, char** argv)
+       setUsageHelp("c USAGE: %s [options] <input-file> <result-output-file>\n\n  where input may be either in plain or gzipped DIMACS.\n");
+ 
+ 
+-#if defined(__linux__)
++#if defined(__linux__) && (defined(__i386__) || defined(__x86_64__))
+         fpu_control_t oldcw, newcw;
+         _FPU_GETCW(oldcw); newcw = (oldcw & ~_FPU_EXTENDED) | _FPU_DOUBLE; _FPU_SETCW(newcw);
+         //printf("c WARNING: for repeatability, setting FPU to use double precision\n");

